authz/rbac_translator.go

```diff
@@ -266,6 +281,69 @@ func parseRules(rules []rule, prefixName string) (map[string]*v3rbacpb.Policy, e
 	return policies, nil
 }
 
+// Parse auditLoggingOptions to the associated RBAC protos. The single
+// auditLoggingOptions results in two different parsed protos, one for the allow
+// policy and one for the deny policy
+func (options *auditLoggingOptions) toProtos() (allow *v3rbacpb.RBAC_AuditLoggingOptions, deny *v3rbacpb.RBAC_AuditLoggingOptions, err error) {
+	allowRbac := v3rbacpb.RBAC_AuditLoggingOptions{}
+	denyRbac := v3rbacpb.RBAC_AuditLoggingOptions{}
```

Comment:
Nit: These could use the same names as the ones in the return value list.

As long as naked returns are not being used, we should be OK to use the same names.

Reply:
Will change